pandas/io/json/json.py

```diff
@@ -263,6 +266,16 @@ def read_json(path_or_buf=None, orient=None, typ='frame', dtype=True,
 
         .. versionadded:: 0.19.0
 
+    chunksize: integer, default None
+        If `lines=True`, how many lines to read into memory at a time.
+        If this is None, the file will be read into memory all at once.
+        Passing a chunksize helps with memory usage, but is slower.
+        Also note this is different from the `chunksize` parameter in
+            `read_csv`, which returns a FileTextReader.
```

Comment:
What is the python 2/3 compat issue with using `yield` ? (it has no `.next` method in python 3, but you should use `next(..)` anyway?) 
(that's how it is done in `read_sql`, so if there is an issue it should be fixed there)